docs/compiling.rst

```diff
@@ -1,3 +1,35 @@
+Installing Libdrizzle Redux
+===========================
+
+Debian
+------
+
+For ``Debian`` based systems running **ubuntu xenial** ``deb`` packages are
+available in the project's **apt** repository at `bintray`_.
+
+To add the **apt** repository to your system follow the instructions given
+when clicking on the **[set up me]!** button.
+
+.. image:: _static/bintray-setmeup.png
+
+The parameter ``{distribution}`` should be ``xenial`` while ``{components}`` can be
+``release`` and/or ``prerelease``. E.g::
+
+   https://dl.bintray.com/sociomantic-tsunami/libdrizzle-redux xenial release prerelease
+
+Then run::
+
+    sudo apt update
+    sudo apt install libdrizzle-redux (for versions on the v5.x branch)
+    sudo apt install libdrizzle-redux6 (for versions on the v6.x branch)
+
+.. _`bintray`: https://bintray.com/sociomantic-tsunami/libdrizzle-redux/libdrizzle-redux
+
+Redhat
+------
+Distribution packages are not available for ``Redhat`` based systems, but ``rpm``
+packages can be generated by running ``make rpm``.
+
```

Comment:
Y the duplication ? Can't you just link to the README if you want to provide the info here ?

Reply:
Yes I guess I could. But think it is better to have the pdf/html manual as self contained docs

Reply:
Then link from the README to `compiling`. A pure copy-paste is always a red flag, and prone to get out of sync at some point.